pkgs/by-name/ne/nerdfetch/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitea
+}:
+stdenv.mkDerivation rec {
+  pname = "nerdfetch";
+  version = "8.1.0";
```

Comment:
Newest version is 8.1.1 and seems to only be released on github https://github.com/ThatOneCalculator/NerdFetch/releases